screens/TabOneScreen.tsx

```diff
@@ -4,30 +4,93 @@ import { StyleSheet } from 'react-native';
 import EditScreenInfo from '../components/EditScreenInfo';
 import { Text, View } from '../components/Themed';
 import { RootTabScreenProps } from '../types';
+import { FlatList, SafeAreaView, StatusBar, TouchableOpacity } from "react-native";
+import { useState } from 'react'
+import { ScreenContainer } from 'react-native-screens';
+import { ScrollView } from 'react-native-gesture-handler';
 
 export default function TabOneScreen({ navigation }: RootTabScreenProps<'TabOne'>) {
+  const [selectedId, setSelectedId] = useState(null);
+
+  const renderItem = ({ item }) => {
+    const backgroundColor = item.id === selectedId ? "#ECECEC" : "#ECECEC";
+    const color = item.id === selectedId ? 'white' : 'black';
+
+    return (
+      <Item
+        item={item}
+        onPress={() => setSelectedId(item.id)}
```

Comment:
Seems like you change the text color to white when you press down. Should the color revert back to black when the user releases finger?